views/admin/requests/index.js

```diff
@@ -239,8 +239,8 @@ var getResults = function(callback) {
     };
 
     var createCSV = function() {
-        var requestFieldNames = ['id','name','address','city','state','zip','phone','email','date created','region'];
-        var requestFields = ['serial','name','address','city','state','zip','phone','email','createdAt','assigned_rc_region'];
+        var requestFieldNames = ['id','name','address','city','state','zip','phone','email','date created','region', 'origin'];
```

Comment:
(See [the other comment](https://github.com/redcross/smoke-alarm-portal/pull/232/files#r134554055), future readers!)